src/go/rpk/pkg/cli/cmd/debug/bundle_linux.go

```diff
@@ -53,7 +55,7 @@ func executeBundle(
 	cl *kgo.Client,
 	admin *admin.AdminAPI,
 	logsSince, logsUntil string,
-	logsLimitBytes int,
+	logsLimitBytes, controllerLogLimitBytes int,
```

Comment:
As follow up, we should probably extract all these args to a dedicated type at this point